src/plugins/saved_objects/public/saved_object/helpers/save_saved_object.ts

```diff
@@ -123,7 +125,9 @@ export async function saveSavedObject(
       chrome.recentlyAccessed.add(
         savedObject.getFullPath(),
         savedObject.title,
-        String(savedObject.id)
+        String(savedObject.id),
+        savedObject.getOpenSearchType(),
+        moment(Date.now()).format('MM/DD/YYYY HH:mm')
```

Comment:
```suggestion
        { type: savedObject.getOpenSearchType(), updateAt: Date.now() }
```
Should be like this?